roles/ipasmartcard_client/README.md

```diff
@@ -0,0 +1,112 @@
+ipasmartcard_client role
+========================
+
+Description
+-----------
+
+This role allows to configure IPA clients for Smart Card authentication.
+
+**Note**: The ansible playbook and role requires deployed IPA clients.
+
+
+Features
+--------
+* Client setup for Smart Card authentication
+
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.5 and up are supported by this role.
+
+
+Supported Distributions
+-----------------------
+
+* RHEL/CentOS 7.6+
+* Fedora 26+
+* Ubuntu
```

Comment:
I suggest removing "Ubuntu", as we'd need, at least, a vars file for it. For example, I haven't found the equivalent for `krb5-pkinit-openssl` package.

Reply:
Yes, ack.